[test_code_review] Connection refused by Responses - the call doesn't match any registered mock

```
>           raise response
E           requests.exceptions.ConnectionError: Connection refused by Responses - the call doesn't match any registered mock.
E           
E           Request: 
E           - GET https://mozphab-phabhost-cdn.devsvcprod.mozaws.net/file/data/ttdssirgxikqwkhapnm7/PHID-FILE-6gufklvfupob535hcqvv/D233024.1743691064.diff
E           
E           Available matches:
E           Passthru prefixes:
E           - https://phabricator.services.mozilla.com/
E           - https://d2mfgivbiy2fiw.cloudfront.net/file/data/
/usr/local/lib/python3.12/site-packages/responses/__init__.py:993: ConnectionError
```

--- a/tests/test_code_review.py
+++ b/tests/test_code_review.py
@@ -7,7 +7,9 @@ from bugbug.tools.code_review import find_comment_scope
 
 def test_find_comment_scope():
     responses.add_passthru("https://phabricator.services.mozilla.com/")
-    responses.add_passthru("https://d2mfgivbiy2fiw.cloudfront.net/file/data/")
+    responses.add_passthru(
+        "https://mozphab-phabhost-cdn.devsvcprod.mozaws.net/file/data/"
+    )
 
     test_data = {
         "https://phabricator.services.mozilla.com/D233024?id=964198": {

